src/components/Header.js

```diff
@@ -7,7 +10,9 @@ import {
 import { MainMenu } from '../MainMenu';
 import FixedMenu from '../FixedMenu';
 
-const Header = ({fix}) => {
+const Header = ({fix, location: location}) => {
+  console.log(location)
+  if (location.pathname === '/') return (<div></div>)
```

Comment:
Where is `Header` being used that this is needed? Also, we should probably not name this `Header` since it has a name conflict with React's `Header`. Never noticed that before.

Reply:
Yes, 'tis new.  It's a hacky way to remove the header but only for the home / splash page.  I believe @ManickYoj will be working on a patch.